test/integration/conftest.py

```diff
@@ -60,12 +67,29 @@ def container_path(tmpdir_factory, container_url, container_name) -> str:
     return slc_path
 
 
-@pytest.fixture(scope="session")
-def saas_connection_params(saas_host, saas_pat, saas_account_id, operational_saas_database_id) -> dict[str, Any]:
-    connection_params = get_connection_params(
-        host=saas_host,
-        account_id=saas_account_id,
-        database_id=operational_saas_database_id,
-        pat=saas_pat,
-    )
-    yield connection_params
+@pytest.fixture(scope='session')
+def db_schema() -> str:
+    return TEST_SCHEMA
+
+
+@pytest.fixture(scope='session')
+def language_alias() -> str:
+    return TEST_LANGUAGE_ALIAS
+
+
+@pytest.fixture(scope='session')
+def deployer_factory(
+        backend_aware_database_params,
+        backend_aware_bucketfs_params,
```

Comment:
Where are those defined?

Reply:
They come from the pytest plugin `pytest-exasol-backend`.